### DISCUSSION TITLE
hardware/controlsurfaces/

### DISCUSSION THREAD

**Discussion by giscus** [CommandPost/FCPCafe#342] [Website Discussions]
# hardware/controlsurfaces/

This page is dedicated to Control Surfaces in Final Cut Pro.

https://fcp.cafe/hardware/controlsurfaces/

**Comment by AdamMannPro** [comment]
Would love to be able to use a PS4 (or even PS5/Xbox) controller to edit, mainly for initial editing like culling using In/Out, insert/append, blade, etc so even the most basic (but time consuming) tasks can be preformed and then using the normal mouse/keyboard for detailed editing

**Reply by latenitefilms** [reply]
You should be able to do this with the help of one of these apps?

https://commandpost.io/control-surfaces/gamepad/
